prep-javascript-functions/main.js

```diff
@@ -0,0 +1,119 @@
+// #1 addTwoNumbers
+// Line 3: A function named addTwoNumbers is being defined with two parameters, num1 and num2.
+function addTwoNumbers(num1, num2) {
+  return num1 + num2;
+}
+/* Line 10: The function addTwoNumbers is being called with the arguments 3 & 6, and the value
+ of that call is being assigned to the variable addTwoNumbersResult. */
+/* Line 11: The log method of the console.object is being called with two arguments, a string &
+ addTwoNumbersResult. */
+const addTwoNumbersResult = addTwoNumbers(3, 6);
+console.log('addTwoNumbers exercise:', addTwoNumbersResult); //9
+
+// #2 convertHoursToMinutes
+// Line 17: A function named convertHoursToMinutes is being defined with one parameter, hours.
+/* Line 18: The result of the expression hours times the number 60 is being assigned to the variable
+ minutes. */
+function convertHoursToMinutes(hours) {
+  const minutes = hours * 60;
+  return minutes;
+}
+/* Line 25: The function convertHoursToMinutes is being called with the argument 4, and the value
+ of that call is being assigned to the variable convertHoursToMinutesResult. */
+/* Line 26: The log method of the console.object is being called with two arguments, a string &
+ convertHoursToMinutesResult. */
+const convertHoursToMinutesResult = convertHoursToMinutes(4);
+console.log('convertHoursToMinutes exercise:', convertHoursToMinutesResult); //240
+
+//#3 getGreeting
+// Line 30: A function named getGreeting is being defined with one parameter, name.
+function getGreeting(name) {
+  const greeting = '' + name + '';
+  return greeting;
+}
+// Line 39: The string Hello is being assigned to the variable greeting.
+/* Line 40: The string value of the variable greeting is being concatenated with ' Tré & the
+ result of that expression is being reassigned to the variable greeting. */
+/* Line 41: The log method of the console.object is being called with two arguments, a string &
+ greeting. */
+let greeting = 'Hello';
+greeting += getGreeting(' Tré');
+console.log('getGreeting exercise:', greeting); //Hello Tré
```

Comment:
Rather than concatenating `greeting` with the result of `getGreeting`, the string should be built in the `getGreeting` function.

So I should be able to just call `getGreeting('Shawn')` and that function will return `'Hello Shawn!'` without any other variables.